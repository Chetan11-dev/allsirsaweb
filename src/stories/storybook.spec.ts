import initStoryshots from '@storybook/addon-storyshots'

it('should ', () => {
    // TODO DONT UNCOMMENT AS REGARD TO https://stackoverflow.com/questions/61706963/typeerror-require-requireactual-is-not-a-function WILL BE FIXED IN STORYBOOK NEXT VERSION WAIT FOR THAT
    // initStoryshots() 
})
