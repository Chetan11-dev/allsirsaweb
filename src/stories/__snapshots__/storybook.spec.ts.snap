// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Product  Correct Product 1`] = `
<div
  className="mt-2 col text-center"
>
  <div
    className="container text-center "
  >
    <i
      className="far fa-images a bg-light p-2 mb-2"
      style={
        Object {
          "fontSize": "90px",
        }
      }
    />
  </div>
  <form
    className=""
  >
    <div
      className="form-group"
    >
      <input
        className="form-control"
        data-test="title"
        name="title"
        onChange={[Function]}
        placeholder="Add Product Title"
        value="Arhar"
      />
    </div>
    <div
      className="form-group"
    >
      <input
        className="form-control"
        data-test="variations"
        name="variations"
        onChange={[Function]}
        placeholder="Add Variations"
        value="20-70,30-90"
      />
      <small
        className="ml-1"
      >
        Eg. 1-50 means 1
        g
         at 50 Rs
      </small>
    </div>
  </form>
  <div
    data-test="variationsBadges"
  >
    <span
      className="m-2 badge badge-pill badge-light"
    >
      20g at 70
    </span>
    <span
      className="m-2 badge badge-pill badge-light"
    >
      30g at 90
    </span>
  </div>
  <div
    className=""
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        g
      </button>
    </div>
  </div>
  <div
    className="p-3"
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        Grocery & Staples
      </button>
    </div>
  </div>
  <div
    className=""
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        Pulses
      </button>
    </div>
  </div>
  <hr />
</div>
`;

exports[`Storyshots Product  Wrong Product 1`] = `
<div
  className="mt-2 col text-center"
>
  <div
    className="container text-center "
  >
    <i
      className="far fa-images a bg-light p-2 mb-2"
      style={
        Object {
          "fontSize": "90px",
        }
      }
    />
  </div>
  <form
    className=""
  >
    <div
      className="form-group"
    >
      <input
        className="form-control"
        data-test="title"
        name="title"
        onChange={[Function]}
        placeholder="Add Product Title"
        value="Arhar"
      />
    </div>
    <div
      className="form-group"
    >
      <input
        className="form-control"
        data-test="variations"
        name="variations"
        onChange={[Function]}
        placeholder="Add Variations"
        value="20-70,30-90"
      />
      <small
        className="ml-1"
      >
        Eg. 1-50 means 1
        g
         at 50 Rs
      </small>
    </div>
  </form>
  <div
    data-test="variationsBadges"
  >
    <span
      className="m-2 badge badge-pill badge-light"
    >
      20g at 70
    </span>
    <span
      className="m-2 badge badge-pill badge-light"
    >
      30g at 90
    </span>
  </div>
  <div
    className=""
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        g
      </button>
    </div>
  </div>
  <div
    className="p-3"
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        Grocery & Staples
      </button>
    </div>
  </div>
  <div
    className=""
  >
    <div
      className="dropdown"
      onKeyDown={[Function]}
    >
      <button
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle btn btn-primary"
        disabled={false}
        id="dropdown-basic-button"
        onClick={[Function]}
        type="button"
      >
        Pulses
      </button>
    </div>
  </div>
  <hr />
</div>
`;
